### DISCUSSION TITLE
Yugabyte DB as a datastore

### DISCUSSION THREAD

**Discussion by sameer-m-dev** [authzed/spicedb#1865] [Q&A] [answered]
Since SpiceDB already supports CockroachDB as one of its datastores, is it also possible to use Yugabyte for the same since by nature both are distributed databases? Please do let me know if this is not possible what could be the reasons for the same, and if SpiceDB has some sort of dependency on CockroachDB

**Comment by vroldanbet** [comment, marked as answer]
@sameer-m-dev being a distributed database is not an enough precondition to work with SpiceDB, key aspect is to be distributed and support strong consistency.

According to [this comment](https://github.com/authzed/spicedb/issues/427#issuecomment-1784678314) in the tracking issue ([and also the docs](https://docs.yugabyte.com/preview/architecture/design-goals/#single-row-linearizability)), Yugabyte does not support Linearizable transactions with multiple rows, so that would be an impediment to implement SpiceDB on top of it.
